awx/main/migrations/0092_v360_webhook_mixin.py

```diff
@@ -39,7 +39,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='workflowjobtemplate',
             name='webhook_service',
-            field=models.CharField(blank=True, choices=[('github', 'GitHub'), ('gitlab', 'GitLab')], help_text='Service that webhook requests will be accepted from', max_length=16),
+            field=models.CharField(blank=True, choices=[('github', 'GitHub'), ('gitlab', 'GitLab'), ('bitbucket', 'Bitbucket')], help_text='Service that webhook requests will be accepted from', max_length=16),
```

Comment:
I do not believe you can rewrite past migrations in this way.

Reply:
But hooks with BitbucketServers was worked.
But callback status to BitbucketServer not working yet...

Reply:
No, this needs to be a new migration.